test/e2e/webdriver/index.js

```diff
@@ -52,6 +52,12 @@ async function setupFetchMocking(driver) {
         if (url.match(/featureFlags$/u)) {
           return { json: async () => clone(mockResponses.swaps.featureFlags) };
         }
+      } else if (
+        url.match(/^https:\/\/(token-api\.airswap-prod\.codefi\.network)/u)
```

Comment:
Can this be `else if(url === ` instead of the regular expression?

Reply:
yes, `else if(url === ` does work, but the above URLs are all using regular expressions for matching. And won't regular expression make it easier to add more params to the URL if needed in the future?